package-lock.json

```diff
@@ -2143,8 +2143,8 @@
       "dev": true
     },
     "frg-ethereum-runners": {
-      "version": "git+https://github.com/ampleforth/frg-ethereum-runners.git#090d0589359a1466c7ad86c64a6a04bc3ef9ac18",
-      "from": "git+https://github.com/ampleforth/frg-ethereum-runners.git#v1.0.0-alpha",
```

Comment:
Should we just use master until we launch? and then assign version numbers?

Reply:
Just a suggestion feel free to ignore :)

Reply:
will do that if we make any more changes. Hopefully we wont